### PR TITLE
Cleaned up experiments.

### PR DESCRIPTION
**Pull request by shokiami**
Cleaned up experiments as discussed.

### CODE REVIEW

File: experiments/experiment3_3.sh [hunk 1]
```diff
@@ -32,45 +32,45 @@ echo "initializing..."
...
           arguments+="$csv_base_path/tmp/$city/train_set$label.csv "
         fi
       done
-      python ../utils/dataset_creator.py "combine" $arguments $csv_base_path/"tmp/subset_"$subsetted_city/$subset/"train_set"$label".csv"
+      python ../utils/dataset_creator.py "combine" "$arguments" "$csv_base_path/tmp/subset_$subsetted_city/$subset/train_set$label.csv"
       rm "$csv_base_path/tmp/subset.csv"
 
       # train model on combined train set
-      python ../train.py $model_name$label $image_base_path/ $csv_base_path/"tmp/subset_"$subsetted_city/$subset/"train_set"$label".csv" $model_name $model_save_folder/$experiment/"subset_"$subsetted_city/$subset/ $num_epochs $crop_size
+      python ../train.py "$model_name$label" "$image_base_path/" "$csv_base_path/tmp/subset_$subsetted_city/$subset/train_set$label.csv" "$model_name" "$model_save_folder/$experiment/subset_$subsetted_city/$subset/" "$num_epochs" "$crop_size"
```
Comment (ThatOneGoat) [inline experiments/experiment3_3.sh:65]: Can we use the same naming convention as in the other experiment scripts for the saved model
`"${experiment}_${model_name}_$label"`

**Review by ThatOneGoat — CHANGES_REQUESTED** [review]
